app/helpers/notifications_helper.rb

```diff
@@ -17,14 +19,17 @@ def generate_notification(user, target_user, team, role)
                        team: team.name,
                        assigned_by_user: user.name)
       end
-      message = "#{I18n.t('search.index.team')} #{team.name}"
     end
 
-    GeneralNotification.send_notifications({
-                                             type: role ? :invite_user_to_team : :remove_user_from_team,
-                                             title: sanitize_input(title),
-                                             message: sanitize_input(message),
-                                             user: target_user
-                                           })
+    GeneralNotification.send_notifications(
+      {
+        type: role ? :invite_user_to_team : :remove_user_from_team,
+        title: sanitize_input(title),
+        subject_id: team.id,
+        subject_class: team.class.name,
+        subject_name: team.respond_to?(:name) && team.name,
+        user: target_user
+      }
+    )
```

Comment:
remove this file and just use this code directly in the controller

Reply:
@artoscinote after our meeting i noticed that this file is being used in 2 controllers, instead removing and duplicating code let's leave it as it is?